### ISSUE TITLE
typo in createListenerMiddleware.mdx file

### ISSUE THREAD

**Issue by yudatadei** [reduxjs/redux-toolkit#2083]
in TypeScript Usage example

```typescript
export const startAppListening = listenerMiddleware.startListening as TypedStartListening<RootState>
export const addAppListener = addListener as  TypedAddListenern<RootState> // should be 'TypedAddListener' not 'TypedAddListenern'
export const addAppListener = addListener as TypedAddListener<RootState> 
````

**Comment by markerikson** [comment]
Thanks, yeah, we'll fix that :)

**Comment by yudatadei** [comment]
This issue was fixed here bc243920831cb6f23b997f1118404b7e9bf2a276
